app.js

```diff
@@ -24,7 +24,7 @@ const bodyParserUrlencodedConfig = () => ({
 
 const init = () => {
   const app = express();
-  const port = config.common.port || 8080;
+  const port = config.common.port || process.env.PORT || 8080;
```

Comment:
`config.common.port` is already an enviroment variable, not need of `process.env.PORT`, add to config file